README.md

```diff
@@ -1,9 +1,10 @@
 # doobie
 
 <img align="right" src="https://cdn.rawgit.com/tpolecat/doobie/series/0.3.x/doobie_logo.svg" height="150px" style="padding-left: 20px"/>
-[![Travis CI](https://travis-ci.org/tpolecat/doobie.svg?branch=master)](https://travis-ci.org/tpolecat/doobie)
+Release:[![Travis CI](https://travis-ci.org/tpolecat/doobie.svg?branch=master)](https://travis-ci.org/tpolecat/doobie)
+0.4.x:[![Travis CI](https://travis-ci.org/tpolecat/doobie.svg?branch=0.4.0-staging)](https://travis-ci.org/tpolecat/doobie)
```

Comment:
I would prefer a single badge where `branch=0.4.x` which will show up as unknown for the moment.